CHANGELOG.md

```diff
@@ -4,13 +4,15 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Querier: the CLI flag `-querier.minimize-ingester-requests` has been moved from "experimental" to "advanced". #7638
```

Comment:
is this intentional?

Reply:
Nope, this slipped through while merging conflicts, thanks!